Source/WebCore/Modules/WebGPU/GPUOutOfMemoryError.h

```diff
@@ -28,6 +28,7 @@
 #include "WebGPUOutOfMemoryError.h"
 #include <wtf/RefCounted.h>
 #include <wtf/RefPtr.h>
+#include <wtf/text/WTFString.h>
```

Comment:
Is this change needed for unified builds or something like that?

Reply:
Yeah it doesn't compile otherwise.